index.bs

```diff
@@ -581,6 +595,26 @@ To <dfn>handle an incoming message</dfn> given a [=WebSocket connection=]
 
 </div>
 
+<div algorithm>
+To <dfn export>emit an event</dfn> given |body| and |related contexts|:
+
+ 1. [=Assert=]: |body| has size 2 and contains "<code>method</code>" and "<code>params</code>".
```

Comment:
Linking https://infra.spec.whatwg.org/#map-size and https://infra.spec.whatwg.org/#map-exists here would be good, also making it clear that *body* is a map.